frontend/lib/src/components/core/Block/ElementNodeRenderer.tsx

```diff
@@ -497,6 +502,20 @@ const RawElementNodeRenderer = (
         />
       )
 
+    case "audioInput": {
+      const audioInputProto = node.element.audioInput as AudioInputProto
+      widgetProps.disabled = widgetProps.disabled || audioInputProto.disabled
```

Comment:
**thought:** It is counterintuitive to me that we mutate `widgetProps`, although I see us doing this in many other places in this file.

My first reaction was that we should not mutate props, but then I realized that `widgetProps` itself is an object defined locally in this function. With that said, we do this in a lot of places throughout this file. I wonder if there's a better abstraction we could build in to prevent us from having to duplicate this logic everywhere. Note that this not something we should do in this PR.

Reply:
💯 I found this a bit strange too, let's put this in the backlog!

Reply:
added it to the Tech Debt DB